### ISSUE TITLE
In-VBE GitHub controls not working

### ISSUE THREAD

**Issue by Zak-Michael-Armstrong** [rubberduck-vba/Rubberduck#832]
On my computer, the in-VBE controls for e.g. commits, branches and other actions have never worked. They've never populated with changes to my code. That is how my SC panel has always looked:

![rd source control - no data](https://cloud.githubusercontent.com/assets/13286014/10454073/d165923e-71a9-11e5-8216-08b1ee59b294.png)

But, they do create and update the repositories as expected. 

My Configuration:

OS: Windows 7 Profesional, service pack 1
Excel: Excel 2013 (Ms Office Home & Business 2013), latest updates
GitHub: Desktop Client, latest version as at 19.08.2015

**Comment by AndrewM-** [comment]
Bug with current build (1.4.3)  apparently, I reported this one a couple of days ago.  A fix is in the works.

**Comment by Zak-Michael-Armstrong** [comment]
Awesome.

**Comment by Zak-Michael-Armstrong** [comment]
If this is a duplicate then, should I close it?

**Comment by Vogel612** [comment]
How I understood it, this part is under significant reworking for 2.0, see also #766 

The issue @AndrewM- is talking about seems to be #822, could you confirm? From what I see, this bug isn't confirmed by the Rubberduck Core Team yet, and the SourceControl features are currently rather stale. This might be a UI-Only issue, though, so there's that...

**Comment by ryanjhodge** [comment]
:+1: I'm also experiencing this bug.

**Comment by AndrewM-** [comment]
@Vogel612 #822 was about how to git works with system like Access and Excel where versioning can also be done by saving the mdb/xls file.  The second issue in #822 was that VBA changes the case of variable names and that breaks diffing.  The issue has been picked up but I reported it on a different repo.  It was confirmed to UI.

**Comment by ckuhn203** [comment]
In the short term, I recommend going back to v1.4 if the source control features are absolutely essential to you. Of course, this means you'll lose many of the fixes introduces in 1.41, 1.42, & 1.43....

**Comment by ckuhn203** [comment]
I think I figured out what happened to break this, and I'm sorry that it went unnoticed. In v1.4, I introduced an IoC container to the project that "wired up" all of the classes for the source control integration, but it was not in the "proper" place in the code. It was farther down in the muck than it should have been, so when @retailcoder started taking inversion of control seriously in v1.4.1 and introduced *another* container where it should have been to begin with, the IoC container for the git integration portion of the project broke.

I sincerely apologize. This is entirely my fault. I got lazy, just wanted to get the feature out the door, and cut some corners to make it happen. I will try my best to get this working again before the end of the year, but in complete honesty, I just don't have the time to work on things that I used to. Also, I don't think I will be patching the main branch, so this fix will have to go out with v2.0, whenever that is.
